pkg/asset/manifests/storageclassconfig_test.go

```diff
@@ -0,0 +1,49 @@
+package manifests
```

Comment:
Can we fix the commit message to indicate these are tests--not validation.

Reply:
Fixed.